modules/introduction/pages/editions.adoc

```diff
@@ -7,39 +7,33 @@
 
-The Community Edition is not subjected to the iterative "test, fix, and verify" quality assurance cycle that is an integral part of the Enterprise Edition release process.
-//The release version of CE is several months behind EE.
-The Community Edition does not include the latest bug fixes.
+The Community Edition license provides the free deployment of Couchbase Community Edition for departmental-scale deployments of up to five node clusters. The Community Edition license has recently been updated to disallow the use of Cross Data Center Replication
+(XDCR), which is now an exclusive Enterprise Edition feature.
+The Community Edition is not subjected to the iterative "test, fix, and verify" quality assurance cycle that is an integral part of the Enterprise Edition release process. The Community Edition does not include the latest bug fixes.
 
 The Community Edition comes with limited concurrency and parallelism and supports a maximum of 4 cores per node.
 
-Documentation, mailing lists, and forums provide support for the Couchbase user community to help troubleshoot issues and answer questions.
+Documentation, mailing lists, and forums support the Couchbase user community to help troubleshoot issues and answer questions.
 
 Open Source Project::
-The Couchbase Server open source project is a platform for innovation.
-Couchbase is committed to open source development and the open source project continues to serve as the foundation for both the Community Edition and the Enterprise Edition.
 
-There are many ways to contribute to the open source project: 
+The Couchbase Server open source project is a platform for innovation. Couchbase is committed to open source development, and the open source project continues to serve as the foundation for both the Community Edition and the Enterprise Edition.
+
+There are many ways to contribute to the open source project:
 
-* Contribute code to our core engine, to our SDKs, connectors and other integration components that help us connect to other products through https://github.com/couchbase[github.com/couchbase^]. Find more details on the https://developer.couchbase.com/open-source-projects/[Open Source Projects^] page.
+* Contribute code to our core engine, our SDKs, connectors and other integration components that help us connect to other products through https://github.com/couchbase[github.com/couchbase^]. Find more details on the https://developer.couchbase.com/open-source-projects/[Open Source Projects^] page.
 * Report issues on our tracking system: https://issues.couchbase.com/projects/MB?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page[JIRA^].
 * https://docs.couchbase.com/home/contribute/index.html[Contribute to the documentation^] either by submitting changes on GitHub or by simply clicking the "Feedback" link in our online documentation.
```

Comment:
This is part of the Antora documentation set, so I think this should be an xref.

```suggestion
* xref:home:contribute:index.adoc[Contribute to the documentation] either by submitting changes on GitHub or by simply clicking the "Feedback" link in our online documentation.
```